plugin.py

```diff
@@ -21,6 +21,10 @@ class LspVolarPlugin(NpmClientHandler):
     server_directory = 'server'
     server_binary_path = os.path.join(server_directory, 'node_modules', '@vue', 'language-server', 'bin', 'vue-language-server.js')
 
+    @classmethod
+    def minimum_node_version(cls) -> Tuple[int, int, int]:
+        return (16, 0, 0)
```

Comment:
This one is deprecated. There is a new method - https://github.com/sublimelsp/LSP-pyright/blob/a1a822b6406386df2459cd8fcb7f3772cecbf9b1/plugin.py#L45-L51